libc/test/include/header-test-template.c

```diff
@@ -2,7 +2,7 @@
  *
  * Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
  * See https://llvm.org/LICENSE.txt for license information.
- * SPDSList-License-Identifier: Apache-2.0 WITH LLVM-exception
+ * SPDXList-License-Identifier: Apache-2.0 WITH LLVM-exception
```

Comment:
I don't know where `SPDSList-License-Identifier` comes from, but with SPDX it should just be `SPDX-License-Identifier` as far as I know. There is not a single occurrence of `SPDXList-License-Identifier` in the repo.

Reply:
This wasn't resolved.

```suggestion
 * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```